FAQ.md

```diff
@@ -231,3 +231,8 @@ Passing an optional command line parameter, `--config-file`, to the Firecracker
 process allows this type of configuration. This parameter must be the path to a
 file that contains the JSON specification that will be used to configure and start
 the microVM. One example of such file can be found at `tests/framework/vm_config.json`.
+
+### Out of Memory Error appears when starting Firecracker
+
+If firecracker exits logging an error with code 12 meaning `Out of memory`, the root 
+cause is not enough memory on the host to run a microVM
```

Comment:
What do you think if we give a mention to how the memory size was configured, or where it defaults to if not configured?

```suggestion
cause is that there is not enough memory on the host, to be used by the Firecracker microVM, as stated in the API request which configured the microVM memory size. If the microVM was not configured in terms of memory size through an API request, the host does not meet the minimum requirement in terms of free memory size, namely 128 MB of free memory, which the microVM defaults to if not configured.
```